samigo/samigo-app/src/java/org/sakaiproject/jsf/util/SamigoJsfTool.java

```diff
@@ -331,14 +330,7 @@ else if (target.indexOf("/jsf/author/publishedSettings") > -1
     	         toolSession.removeAttribute("SENT_TO_FILEPICKER_HELPER");
       }
 
-      // case 4: create new mail, then set
-		// emailBean.attachmentList = filepicker list
-      else if (target.indexOf("/jsf/evaluation/createNewEmail") > -1
-				&& ("true").equals(toolSession.getAttribute("SENT_TO_FILEPICKER_HELPER"))) {
-			EmailBean bean = (EmailBean) ContextUtil.lookupBeanFromExternalServlet("email", req, res);
-			bean.prepareAttachment();
-			toolSession.removeAttribute("SENT_TO_FILEPICKER_HELPER");
-		}
+      // case 4: removed legacy email functionality
```

Comment:
This comment feels unnecessary since it won't make sense as the code is gone.

Reply:
agreed